### DISCUSSION TITLE
possible to custom container for pagination ?

### DISCUSSION THREAD

**Discussion by d-strygwyr** [nolimits4web/swiper#6522] [Q&A]
possible to custom container pagination to like this.
![image](https://user-images.githubusercontent.com/49369600/227717657-38434abb-414e-40ee-9a4d-11251657f60b.png)

if i inspect the element on pagination it looks like this 
![image](https://user-images.githubusercontent.com/49369600/227717784-7afd9a2d-b754-4a40-b213-4bfa73992715.png)

after div `swiper-pagination` and then the bullet element, is it possible to add some wrapper before the bullet element? 

like 
```html
<div class='swipper-pagination'>
  <div class='some-wrapper'>
       ....bullet element
 </div>
</div>
```

**Comment by VIPStephan** [comment]
You don’t need another wrapper, you can just style the pagination container to your liking with CSS, too. Alternatively, you wouldn’t add a wrapper inside the pagination container, you would put the pagination container inside any custom (hard-coded) wrapper and style that wrapper.
